apps/libsignpost/signbus_io_interface.c

```diff
@@ -422,7 +427,7 @@ static void signbus_iterate_slave_read(void) {
 
         } else {
             // perform the callback!
-            slave_read_callback(SUCCESS);
+            slave_read_callback(0);
```

Comment:
SUCCESS is a tock thing - I was trying to make signbus platform agnostic. We'll have to decide on a set of error/success codes.